components/TxForm.tsx

```diff
@@ -29,6 +52,108 @@ export const TxForm = () => {
   // TODO: Placeholder notification for shielding
   //
   const { notifyUser } = useNotifications();
+  let abi = [
+    "function transfer(address,uint256) returns (bool)",
+    "function approve(address,uint256) returns (bool)",
+  ];
```

Comment:
Can replace this with `import { erc20ABI } from '@wagmi/core'`: https://wagmi.sh/core/constants/abis

Reply:
done https://github.com/ScopeLift/railgun-shielder/blob/e933fe01d57c55ab1872747249001094eb43d487/components/TxForm.tsx#L21